nuitka-build.zsh

```diff
@@ -1,14 +1,14 @@
-python -m nuitka \
+python3.11 -m nuitka \
     --company-name=AMANCA\ SOFTWARE \
     --product-name=Village\ Kids\ Pager \
-    --product-version=1.0.0 \
+    --product-version=1.1.0 \
     --copyright=Angelo\ Manca \
     --standalone \
     --enable-plugin=pyside6 \
     --macos-app-name=Village\ Kids\ Pager \
     --macos-app-mode=gui \
-    --macos-signed-app-name=com.amanca.VKPager \
```

Comment:
we should still have a bundle name, mac can complain when one isnt provided if you move the install to another device iirc

Reply:
I was having issues building without the signing cert, I might have just commented out anything with "sign*" in it. I'll try building with that uncommented.